### ISSUE TITLE
Error Report, after deleting the last few files

### ISSUE THREAD

**Issue by armarti** [arsenetar/dupeguru#339]
FIRST ERROR POPUP:

```
Application Name: dupeGuru Music Edition
Version: 6.8.1

Traceback (most recent call last):
  File "/usr/share/dupeguru_me/qtlib/table.py", line 85, in data
    return self._getData(row, column, role)
  File "/usr/share/dupeguru_me/qt/base/results_model.py", line 35, in _getData
    elif row.is_cell_delta(column.name):
  File "/usr/share/dupeguru_me/core/gui/result_table.py", line 43, in is_cell_delta
    ref_info = self._group.ref.get_display_info(group=self._group, delta=False)
AttributeError: 'NoneType' object has no attribute 'get_display_info'
```

SECOND ERROR POPUP:

```
Application Name: dupeGuru Music Edition
Version: 6.8.1

Traceback (most recent call last):
  File "/usr/share/dupeguru_me/qtlib/table.py", line 83, in data
    row = self.model[index.row()]
  File "/usr/share/dupeguru_me/hscommon/gui/table.py", line 42, in __getitem__
    return self._rows.__getitem__(key)
IndexError: list index out of range
```

THIRD ERROR POPUP:

```
Application Name: dupeGuru Music Edition
Version: 6.8.1

Traceback (most recent call last):
  File "/usr/share/dupeguru_me/qtlib/table.py", line 83, in data
    row = self.model[index.row()]
  File "/usr/share/dupeguru_me/hscommon/gui/table.py", line 42, in __getitem__
    return self._rows.__getitem__(key)
IndexError: list index out of range
```

FOURTH:

```
Application Name: dupeGuru Music Edition
Version: 6.8.1

Traceback (most recent call last):
  File "/usr/share/dupeguru_me/qtlib/table.py", line 83, in data
    row = self.model[index.row()]
  File "/usr/share/dupeguru_me/hscommon/gui/table.py", line 42, in __getitem__
    return self._rows.__getitem__(key)
IndexError: list index out of range
```

---

There were a fifth and sixth but I didn't get them.

Great apps!

-- David

**Comment by hsoft** [comment]
Might be related to #309, but maybe not. All errors seem to be byproducts of the first crash of the list.

**Comment by smileleaf** [comment]
I compared between location A (Ref) and Location B. Then I deleted files from location B without problems. I then copied other files to location B (on the same run, meaning I didn't close the software). Deleted the second matches but then got this message (the files are deleted): 

Application Name: dupeGuru
Version: 3.9.1

Traceback (most recent call last):
  File "C:\src\dupeguru\qtlib\table.py", line 85, in data
  File "C:\src\dupeguru\qt\base\results_model.py", line 35, in _getData
  File "C:\src\dupeguru\core\gui\result_table.py", line 43, in is_cell_delta
AttributeError: 'NoneType' object has no attribute 'get_display_info'
